playground/milvus/MilvusPlayground.ApiService/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "",
+      "applicationUrl": "http://localhost:5456",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "https": {
```

Comment:
add `generate-manifest` profile and make sure you check-in the manifest.

Reply:
Can we automate this somehow?

Do we really need a `generate-manifest` profile? Why can't we just have a script that `dotnet run -- ...` all the playground projects with the correct args? Then we don't need to duplicate the args in every playground app.